kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/FieldNameSanitizer.java

```diff
@@ -24,6 +29,9 @@ public static String sanitizeName(String name) {
     return map.entrySet().stream().collect(Collectors.toMap(
         (entry) -> sanitizeName(entry.getKey()),
         (entry) -> {
+          if (entry.getValue() == null) {
+            throw new ConnectException("Record values cannot be null");
```

Comment:
I don't think this is true--BigQuery supports `NULLABLE` columns, so if a null entry here corresponds to that, then we should permit this.

Reply:
It seems that the exception in the stack trace corresponds to the exception that I get when I test for null values without this null check.
```
Caused by: java.lang.NullPointerException
	at java.util.HashMap.merge(HashMap.java:1225)
	at java.util.stream.Collectors.lambda$toMap$58(Collectors.java:1320)
	at java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169)
	at java.util.HashMap$EntrySpliterator.forEachRemaining(HashMap.java:1699)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
	at com.wepay.kafka.connect.bigquery.utils.FieldNameSanitizer.replaceInvalidKeys(FieldNameSanitizer.java:29)
	at com.wepay.kafka.connect.bigquery.utils.FieldNameSanitizerTest.testNullValue(FieldNameSanitizerTest.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:19)
	... 15 more
```

Reply:
I think we're conflating Java's lack of support for null values in `Collectors::toMap` with invalid data. It's fine if there are null entries in this map in terms of what gets sent to BigQuery; it just breaks because we're using streams here. We might consider rewriting this with a vanilla for-loop.

Reply:
Is it not true that there is some config that allows for field sanitizing to false?

Reply:
Alternatively, we can not use the `collect` method if we want to allow sanitizing with null values?

Reply:
Right--I think the way to do that (avoid usage of `Collectors::toMap`) is to use a vanilla for-loop instead of Java 8 streams. We shouldn't tell people to disable sanitization unless that's the only possible workaround, but I think we should be alright if we just avoid using streams.

Reply:
Alright, I think this is the way to go as well.